rocm-cmake/PKGBUILD

```diff
@@ -20,4 +20,6 @@ build() {
 
 package() {
   make DESTDIR="$pkgdir" install
+
+  install -Dm644 "$_dirname/LICENSE" "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```

Comment:
Why is rocm-cmake changed in this PR?

Reply:
Weird, I don't remember doing that, but it should be cherry picked to master anyway.